### PR TITLE
DOCS-82 - The documentation related to annotation in the network poli…

### PR DESCRIPTION
**Pull request by rprav007**
…cies page needs to be updated for 1.7

### CODE REVIEW

File: Documentation/network-policies.md [hunk 1]
```diff
@@ -13,14 +13,23 @@ To keep your cluster safer, it's strongly recommended to enable network policies
 In this example we are closing all the inbound communication on the namespace monitoring and just allowing the necessary traffic.
 **This example are only tested with calico provider.**
```
Comment (zbwright) [inline Documentation/network-policies.md]: This example will close all inbound communication on the namespace monitoring, and allow only necessary traffic. 
**This example has been tested only with the Calico provider.**

File: Documentation/network-policies.md [hunk 2]
```diff
@@ -13,14 +13,23 @@ To keep your cluster safer, it's strongly recommended to enable network policies
...
 
-Follow the steps [here](http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/) to install calico, also dont' forget to [enable network policy](https://kubernetes.io/docs/user-guide/networkpolicies) in you k8s cluster.
+Follow the steps [here](http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/) to install calico, also dont' forget to [enable network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) in you k8s cluster.
```
Comment (zbwright) [inline Documentation/network-policies.md]: First, follow the instructions to [add Calico to an existing Kubernetes cluster][add-calico]. Be certain to [enable network policy][enable-policy] in the cluster.

the bottom of the page gets these links:
[add-calico]: http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/
[enable-policy]: https://kubernetes.io/docs/user-guide/networkpolicies/

question: which policy?

File: Documentation/network-policies.md [hunk 3]
```diff
@@ -13,14 +13,23 @@ To keep your cluster safer, it's strongly recommended to enable network policies
...
 
-Once you've done that, you should annotate your namespace to deny all the ingress (inbound) traffic.
+Once you've done that, you should use the following configuration to deny all the ingress (inbound) traffic.
```
Comment (zbwright) [inline Documentation/network-policies.md]: Next, use the following configuration to deny all the ingress (inbound) traffic.

**Comment by brancz** [comment]
Could you rebase this PR, and take care of the comments? :slightly_smiling_face:

**Comment by zbwright** [comment]
I don't have edit rights in this repo, or I totally would.

**Comment by zbwright** [comment]
@rprav007 @brancz tiny bump. can we get this merged this week?

File: Documentation/network-policies.md [hunk 4]
```diff
@@ -13,14 +13,23 @@ To keep your cluster safer, it's strongly recommended to enable network policies
...
+metadata:
+name: default-deny-all
+namespace: default
+spec:
+podSelector:
+matchLabels:
 ```
+Save the config file as default-deny-all.yaml and apply the configuration to the cluster using
+
+```kubectl apply -f <path to config file>/default-deny-all.yaml```
```
Comment (brancz) [inline Documentation/network-policies.md]: This needs line breaks after the starting rune and before then enduring one. Our tests are complaining about an unbalanced section here.

**Comment by rprav007** [comment]
@zbwright @brancz, all comments addressed.

**Comment by brancz** [comment]
Tests passing and doc changes lgtm from my perspective.

@zbwright feel free to merge if this lgty

**Comment by zbwright** [comment]
thank you both - I don't have merge privileges to this repo. all yours, @brancz

**Comment by brancz** [comment]
Thanks!

**Pull request merged**